pkg/compactor/compactor_test.go

```diff
@@ -1758,18 +1762,13 @@ func TestMultitenantCompactor_ShouldFailCompactionOnTimeout(t *testing.T) {
 	// Set ObservePeriod to longer than the timeout period to mock a timeout while waiting on ring to become ACTIVE
 	cfg.ShardingRing.ObservePeriod = time.Second * 10
 
-	c, _, _, logs, _ := prepare(t, cfg, bucketClient)
+	c, _, _, _, _ := prepare(t, cfg, bucketClient)
 
 	// Try to start the compactor with a bad consul kv-store. The
 	err := services.StartAndAwaitRunning(context.Background(), c)
 
 	// Assert that the compactor timed out
-	assert.Equal(t, context.DeadlineExceeded, err)
-
-	assert.ElementsMatch(t, []string{
-		`level=info component=compactor msg="waiting until compactor is ACTIVE in the ring"`,
-		`level=error component=compactor msg="compactor failed to become ACTIVE in the ring" err="context deadline exceeded"`,
```

Comment:
Why did you delete this log in the `starting()`? I was handy to check the actual reason why it failed to start.

Reply:
Because we no longer emit these log messages in the test now that we've added a timeout to `ringSubservices.AwaitHealthy` and `ring.WaitInstanceState` - these messages only happen after `ringSubservices.AwaitHealthy` has run. I could rearrange things so they're emitted but I'm not a fan of testing log messages.

Reply:
Ok keep it as is and let's move on 😉